packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -846,6 +846,9 @@ class _PlatformViewLinkState extends State<PlatformViewLink> {
 
     if (widget.viewType != oldWidget.viewType) {
       _controller?.dispose();
+      // the _surface has to be recreated as its controller is disposed
```

Comment:
nit: please use complete sentences starting with a capital letter and ending with a `.`.

Reply:
done :-)